### ISSUE TITLE
Regression 0.5.80 -> 0.5.81: CS implicit 'civet coffeeCompat' header

### ISSUE THREAD

**Issue by greghuc** [DanielXMoore/Civet#436]
Bit of a weird regression. I'm running civet over a bunch of coffeescript files, and depending on civet adding an implicit "civet coffeeCompat" at the top of each coffeescript file. Something changed between 0.5.80 -> 0.5.81 which has broken this (but not on all CS files). I'm seeing a bunch of parse errors.

To repo:

civet -c test.coffee, for test.coffee of:
```
MEANING_OF_LIFE = 42

module.exports = MEANING_OF_LIFE
```
gets me on 0.5.81 (but fine on 0.5.80)
```
ParseError: test.coffee:3:1 Failed to parse
Expected:
	NonNewlineWhitespace /[ \t]+/
	NonNewlineWhitespace "\\\\"
	JSSingleLineComment /\/\/(?!\/)[^\r\n]*/
	CoffeeSingleLineComment /#(?!##(?!#))([^\r\n]*)/
	JSMultiLineComment "/*"
	CoffeeHereCommentStart /###(?!#)/
	EOL /\r\n|\n|\r|$/
	Whitespace /[\s]+/
	AssignmentOpSymbol "**="
	AssignmentOpSymbol "*="
	AssignmentOpSymbol "/="
	AssignmentOpSymbol "%="
	AssignmentOpSymbol "+="
	AssignmentOpSymbol "-="
	AssignmentOpSymbol "<<="
	AssignmentOpSymbol ">>>="
	AssignmentOpSymbol ">>="
	AssignmentOpSymbol "&&="
	AssignmentOpSymbol "&="
	AssignmentOpSymbol "^="
	AssignmentOpSymbol "||="
	AssignmentOpSymbol "|="
	AssignmentOpSymbol "??="
	AssignmentOpSymbol "?="
	AssignmentOpSymbol "="
	CoffeeWordAssignmentOp "and="
	CoffeeWordAssignmentOp "or="
	As "as"
	Satisfies "satisfies"
Found: "module.exports"

    at validate (/vagrant/node_modules/@danielx/civet/dist/main.js:488:25)
    at parse (/vagrant/node_modules/@danielx/civet/dist/main.js:515:18)
    at compile (/vagrant/node_modules/@danielx/civet/dist/main.js:23067:15)
    at cli (/vagrant/node_modules/@danielx/civet/dist/civet:281:40) {
  filename: 'test.coffee',
  line: 3,
  column: 1,
  offset: 44
}
```
but adding an explicit 'civet coffeeCompat' to test.coffee works fine:
```
"civet coffeeCompat"

MEANING_OF_LIFE = 42

module.exports = MEANING_OF_LIFE
```

**Comment by greghuc** [comment]
Same error on civet latest 0.5.83

**Comment by STRd6** [comment]
Looks like it's due to this commit: https://github.com/DanielXMoore/Civet/commit/0711de47fc23176c37d45fc687c8ea30d79b0057

**Comment by STRd6** [comment]
Fixed in v0.5.84.

**Comment by greghuc** [comment]
Great. All Coffeescript code is back to compiling, and latest civet forced some previously undiscovered indentation fixes!
